common/test/run-json.c

```diff
@@ -4,7 +4,41 @@
 /* AUTOGENERATED MOCKS START */
 /* AUTOGENERATED MOCKS END */
 
-int main(void)
+
+// issue #577
+
+static void do_json_tok_bitcoin_amount(const char* val, uint64_t expected)
+{
+	uint64_t amount;
+	jsmntok_t tok;
+
+	tok.start = 0;
+	tok.end = strlen(val);
+
+	fprintf(stderr, "do_json_tok_bitcoin_amount(\"%s\", %lu): ", val, expected);
```

Comment:
`"%lu"` fails on 32bit archs, please use `"...%"PRIu64"..." instead, which translates into an arch specific format string

Reply:
right, I will change it.

Reply:
Yeah, sorry, I was eager to get your change merged :-)